docs/requirements.txt

```diff
@@ -1,2 +1,3 @@
 sphinx-markdown-tables==0.0.16
 sphinx-rtd-theme==1.0.0
+urllib3<2
```

Comment:
@visionjo Added this line here. My changes to the .readthedocs.yaml didn't seem to have any impact (didn't fix the problem).

Reply:
@SerenaCaplins nice find. Making its own PR for the sake of being able to trace it back in case.

I will add you as reviewer to pass it through so we can update these other branches (PRs) and get things working again.